postgres/problems/problem_13.md

```
@@ -1,20 +1,19 @@
+**難易度**：★★  
+**学習トピック**：SELECT, WHERE, サブクエリ
 
 ## 説明
-商品別 (`product_id`) に日次売上額を集計した上で、
-**直近 7 日 (当日を含む 7 日間) の移動平均** を求めてください。
+`order_items` テーブルに存在しない `product_id` を持つ商品を `products` テーブルから取得してください。
 
 ## 制約
-* ウィンドウフレーム `ROWS BETWEEN 6 PRECEDING AND CURRENT ROW` を使用すること
-* 結果は `sales_day` 昇順で並べること
+* WHERE 内で NOT IN を使用すること
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)